Source/Core/DolphinQt/Debugger/MemoryViewWidget.cpp

```diff
@@ -102,83 +164,81 @@ void MemoryViewWidget::Update()
 
   for (int i = 0; i < rows; i++)
   {
-    u32 addr = m_address - ((rowCount() / 2) * 16) + i * 16;
+    u32 row_address = address - ((rowCount() / 2) * m_bytes_per_row) + i * m_bytes_per_row;
 
     auto* bp_item = new QTableWidgetItem;
     bp_item->setFlags(Qt::ItemIsEnabled);
-    bp_item->setData(Qt::UserRole, addr);
+    bp_item->setData(Qt::UserRole, row_address);
 
     setItem(i, 0, bp_item);
 
-    auto* addr_item = new QTableWidgetItem(QStringLiteral("%1").arg(addr, 8, 16, QLatin1Char('0')));
+    auto* row_item =
+        new QTableWidgetItem(QStringLiteral("%1").arg(row_address, 8, 16, QLatin1Char('0')));
 
-    addr_item->setData(Qt::UserRole, addr);
-    addr_item->setFlags(Qt::ItemIsEnabled | Qt::ItemIsSelectable);
+    row_item->setData(Qt::UserRole, row_address);
+    row_item->setFlags(Qt::ItemIsEnabled | Qt::ItemIsSelectable);
 
-    setItem(i, 1, addr_item);
+    setItem(i, 1, row_item);
 
-    if (addr == m_address)
-      addr_item->setSelected(true);
+    if (row_address == address)
+      row_item->setSelected(true);
 
-    if (Core::GetState() != Core::State::Paused || !accessors->IsValidAddress(addr))
+    if (Core::GetState() != Core::State::Paused || !accessors->IsValidAddress(row_address))
     {
       for (int c = 2; c < columnCount(); c++)
       {
         auto* item = new QTableWidgetItem(QStringLiteral("-"));
         item->setFlags(Qt::ItemIsEnabled);
-        item->setData(Qt::UserRole, addr);
+        item->setData(Qt::UserRole, row_address);
 
         setItem(i, c, item);
       }
 
       continue;
     }
 
-    if (m_address_space == AddressSpace::Type::Effective)
-    {
-      auto* description_item = new QTableWidgetItem(
-          QString::fromStdString(PowerPC::debug_interface.GetDescription(addr)));
-
-      description_item->setForeground(Qt::blue);
-      description_item->setFlags(Qt::ItemIsEnabled);
-
-      setItem(i, columnCount() - 1, description_item);
-    }
```

Comment:
Why did you remove this?

Reply:
It  places symbol data in a column that is overwritten by normal value data.  Am I missing some case where it actually appears?

Reply:
Mmm you're right but this is clearly trying to do something useful. Whatever, we can fix it later.

Reply:
Maybe two column mode can have address + symbol.  The symbols are what you get from codewidget.  They could also be used as bookmarks if there were additional UI stuff to make and edit them in memorywidget.